server/test/controllers/monitoring/MetricsControllerTest.java

```diff
@@ -62,7 +62,7 @@ public void getMetrics_returnsMetricData() {
     assertThat(metricsContent).contains("ebean_queries_total_latency_micros");
     assertThat(metricsContent).contains(getEbeanCountName("models.ProgramModel"));
     assertThat(metricsContent).contains(getEbeanCountName("models.Question"));
-    assertThat(metricsContent).contains(getEbeanCountName("Version.byId"));
+    assertThat(metricsContent).contains(getEbeanCountName("VersionModel.byId"));
     assertThat(metricsContent).contains("location=\"VersionRepository.getActiveVersion");
```

Comment:
This statement causes the test to fail. Seems to indicate there's something else to refactor in the codebase?

Reply:
Thank you! Yeah - just fixed the MetricsController and this now passes.